### ISSUE TITLE
Website User Database

### ISSUE THREAD

**Issue by TheSaghbini** [WeAreBini/Bini-Site#7]
Can you see if WordPress can have its own database to hold its own database of users and is it easily accessible? Or is it better to use Azure DB to store all users, and if we use Azure DB is integratable to WordPress?

**Comment by damanveen** [comment]
Yes, WordPress can have its own database to hold its own database of users. In fact, WordPress comes with its own built-in user database which is managed through the WordPress Dashboard. By default, WordPress uses the MySQL database to store its data, including user information.

The user database in WordPress can be easily accessed through the WordPress API or by using MySQL commands. Additionally, there are a number of plugins available for WordPress that provide additional functionality for managing user data, such as user roles and permissions, custom user fields, and more.

As for using Azure DB to store user data, it is definitely possible and can be integrated with WordPress. Azure DB is a cloud-based relational database service offered by Microsoft, and it can be used to store a wide range of data, including user information.

**Comment by damanveen** [comment]
To integrate Azure DB with WordPress, you can use a plugin like Azure Database for MySQL or Azure Database for PostgreSQL, depending on which database type you're using. Here are the general steps to follow:

Create an Azure DB instance: If you haven't already, create an instance of Azure DB that will hold your WordPress data. You'll need to specify the database type, server name, and login credentials.

Install and activate the Azure Database plugin: In the WordPress Dashboard, go to Plugins > Add New and search for the "Azure Database" plugin. Install and activate the plugin.

Connect to your Azure DB instance: In the WordPress Dashboard, go to Settings > Azure Database and enter the connection details for your Azure DB instance, including the server name, database name, and login credentials.

Test the connection: Click the "Test Connection" button to verify that WordPress can connect to your Azure DB instance.

Migrate your WordPress data: If you're starting with a fresh Azure DB instance, you can skip this step. But if you want to migrate your existing WordPress data to Azure DB, you'll need to use a tool like the WP Migrate DB plugin. This plugin allows you to export your WordPress data as an SQL file, which you can then import into Azure DB using a tool like the MySQL command-line interface.

Once you've completed these steps, your WordPress site should be using Azure DB as its backend database. Keep in mind that there may be some differences in behavior between using WordPress's built-in database and using Azure DB, so you may need to adjust your site's settings and plugins accordingly.

**Comment by TheSaghbini** [comment]
Perfect, is there Firebase DB connection, I'm thinking to use the same as mobile, so people have the same info on mobile and web, cause I've been using Firebase lol
